frontend/src/components/Shared/CrossingMapPage/CrossingSidebarNearbyCrossingItem.scss

```diff
@@ -0,0 +1,65 @@
+@import 'css/mixins/flexbox';
```

Comment:
Did we decide to move away from the flexbox mixins? This looks like an easy spot to remove a ref to it.

Reply:
There's still a flexbox mixin here. I'll submit another PR with everything.